coordinator/core/core.go

```diff
@@ -561,6 +554,28 @@ func (c *Core) generateCertificateForSecret(secret manifest.Secret, parentCertif
 	return secret, nil
 }
 
+// GenerateUsersFromManifest creates users and permissions from a map of manifest.User
+func GenerateUsersFromManifest(rawUsers map[string]manifest.User) ([]*user.User, error) {
+	// Parse & write X.509 user data from manifest
+	users := make([]*user.User, 0, len(rawUsers))
+	for name, userData := range rawUsers {
+		block, _ := pem.Decode([]byte(userData.Certificate))
+		if block == nil {
+			return nil, fmt.Errorf("received invalid certificate for user %s", name)
+		}
+		cert, err := x509.ParseCertificate(block.Bytes)
+		if err != nil {
+			return nil, err
+		}
+		newUser := user.NewUser(name, cert)
+		newUser.Assign(user.NewPermission(user.PermissionSetSecret, userData.WriteSecrets))
```

Comment:
```suggestion
		newUser.Assign(user.NewPermission(user.PermissionWriteSecret, userData.WriteSecrets))
```
shouldn't this also be renamed then?